dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/Java8IncompatibleReferenceCheck.java

```diff
@@ -35,20 +35,20 @@
 import java.util.logging.Logger;
 import org.eclipse.aether.artifact.Artifact;
 
-public class BadJdkReferenceChecker {
+public class Java8IncompatibleReferenceCheck {
```

Comment:
JavaDoc would help clarify this.

Reply:
Added Javadoc. I'll prepare another document (wiki) to explain how to use this tool.